### PR TITLE
Improves error handling and adds timestamp to log messages

### PR DESCRIPTION
**Pull request by lorengordon**

### CODE REVIEW

File: modules/rdsh/ra_rdsh_autoscale_internal_lb.template.cfn.yaml [hunk 1]
```diff
@@ -298,9 +298,11 @@ Resources:
   AutoScalingGroup:
     CreationPolicy:
       ResourceSignal:
-        Count: !Ref DesiredCapacity
+        Count: 1
         Timeout: PT85M
     Properties:
+      Cooldown: 0
+      DefaultInstanceWarmup: 0
```
Comment (eemperor) [inline modules/rdsh/ra_rdsh_autoscale_internal_lb.template.cfn.yaml:305]: Seems like `DefaultInstanceWarmup` isn't supported by Cloudformation:

https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-as-group.html#cfn-as-group-defaultinstancewarmup

File: scripts/configure-rdsh.ps1 [hunk 2]
```diff
@@ -332,40 +333,43 @@ Invoke-Expression "scoop install --global $($ScoopPackages -join ' ')"
 Invoke-Expression "scoop list" -OutVariable ScoopList
 foreach ($Package in $ScoopPackages + @("git", "7zip")) {
     if (-not $ScoopList.Name.Contains($Package)) {
-        Write-Error "ERROR: Failed to install Scoop package: $Package"
+        Write-Verbose "[$(get-date -format o)]: Retrying install of failed Scoop package: $Package"
+        Invoke-RetryCommand -Command "scoop install --global $Package"
```
Comment (eemperor) [inline scripts/configure-rdsh.ps1:337]: Don't you want to re-check if the install retry succeeded and error if it didn't?

File: modules/rdsh/ra_rdsh_autoscale_internal_lb.template.cfn.yaml (hunk 1 above)
Comment (lorengordon) [reply modules/rdsh/ra_rdsh_autoscale_internal_lb.template.cfn.yaml:305]: It is though! 

* https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-as-group.html#cfn-as-group-defaultinstancewarmup

It's just recently added and lots of things haven't caught up yet.

File: scripts/configure-rdsh.ps1 (hunk 2 above)
Comment (lorengordon) [reply scripts/configure-rdsh.ps1:337]: The Invoke-RetryCommand will do that automatically. It retries 5 times by default, and if they all fail then it exits with error.

File: modules/rdsh/ra_rdsh_autoscale_internal_lb.template.cfn.yaml (hunk 1 above)
Comment (eemperor) [reply modules/rdsh/ra_rdsh_autoscale_internal_lb.template.cfn.yaml:305]: Ok.

**Review by eemperor — DISMISSED** [review]
Looks good to me.

**Review by eemperor — APPROVED** [review]
:shipit:

**Pull request merged**